templates/common/storybook/package.json

```diff
@@ -3,23 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "storybook:install": "cd storybook && yarn",
-    "storybook:run": "cd storybook && yarn start-storybook",
-    "storybook:build": "cd storybook && yarn build-storybook"
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook -c .storybook -o ./storybook-output"
   },
   "dependencies": {
     "@babel/core": "^7.15.5",
     "@babel/preset-react": "^7.12.10",
+    "@babel/preset-typescript": "^7.16.7",
     "@storybook/addon-a11y": "^6.3.8",
     "@storybook/addon-actions": "^6.3.8",
     "@storybook/addon-docs": "^6.3.8",
     "@storybook/addon-knobs": "^6.3.1",
     "@storybook/addon-links": "^6.3.8",
     "@storybook/addon-viewport": "^6.3.8",
-    "@storybook/cli": "^6.4.14",
     "@storybook/react": "^6.3.8",
     "babel-loader": "^8.2.2",
     "css-loader": "^3.6.0",
+    "react-storybook-addon-chapters": "^3.1.7",
```

Comment:
Let's not include an unofficial packages, they bring breaking changes from time to time.